scripts/swtpm_sim.test

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Run tests against SWTPM simulator
+# Setting TESTS to a list of tests to run
+#   TESTS=./examples/native/native_test\ ./examples/wrap/wrap_test ./scripts/swtpm_sim.test
+#
+
+# set -x
+
+TOP_DIR=$(realpath $(dirname $0)/..)
+
+die() {
+    echo $* >&2
+    exit 1
+}
+
+build() {
+    if [ ! -d "$SWTPM_DIR" ]; then
+        echo "Cloning SWTPM from $SWTPM_GIT_URL to $SWTPM_DIR";
+        git clone $SWTPM_GIT_URL $SWTPM_DIR || die "unable to clone $SWTPM_GIT_URL for simualtor";
+    fi
+    echo "Building SWTPM: $(git -C "$SWTPM_DIR" describe --long)";
+    make -C $SWTPM_DIR/src > /dev/null || die "Failed to build $SWTPM_DIR";
```

Comment:
On the Mac this uses the `make -f makefile.mac` file. Can you update, so it builds for Mac too?

```
if [[ "$OSTYPE" == "darwin"* ]]; then
    make -f makefile.mac
fi
```

Reply:
Also might consider "-j4" or something to boost build time.

Reply:
Good idea. I can't test on Mac. Hope my approach works.